docs/usage/getting-started/20-running.md

```diff
@@ -0,0 +1,109 @@
+# Running Renovate
+
+As a Renovate end user, there are two main categories of use:
+
+- You self-host Renovate, e.g. by running the pre-built Docker image, or
+- Someone else is hosting Renovate, and you install/configure it for the repositories you choose
+
+If someone else is hosting Renovate for you, or you are using the WhiteSource Renovate App on GitHub, then you can skip ahead to the Installing Renovate into Repositories section.
+
+## Self-Hosting Renovate
+
+Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+
+- You provide infrastructure for it to run on,
+- You provision its global config,
+- You ensure it's running regularly
+
+### Available distributions
+
+#### npm Package (CLI)
+
+Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
+You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+If you need to support any non-npm lock files like Bundler, Go Modules or Poetry then you'll need to make sure they are pre-installed in the same environment alongside Renovate.
+
+The `renovate` npm package is compatible with all of Renovate's supported platforms.
+
+#### Docker Image
+
+The `renovate` npm package is also distributed in pre-built Node.js images on Docker Hub (`renovate/renovate`).
+
+The `slim` image contains only Node.js so works if either:
+
+- You do not require any additional package managers, or
+- You map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when necessary
+
+The "full" image (which `latest` defaults to) contains every package manager which Renovate supports already preinstalled.
+This approach works best for many, but does have the following downsides:
+
+- It only contains _one_ version of each language/manager - usually the latest
+- It's several gigabytes in size
+
+The `renovate/renovate` Docker images are compatible with all of Renovate's supported platforms.
+
+#### GitHub Action
+
+Renovate's npm tool is also provided as a pre-prepared GitHub Action on [`renovatebot/github-action`](https://github.com/renovatebot/github-action).
+Details on how to use it can be found in the repository.
+
+#### GitLab Runner
```

Comment:
```suggestion
#### GitLab runner
```

Is `Runner` a proper noun, does it refer to a specific capability of GitLab? If it's a proper noun, you can skip this suggestion.

Reply:
![image](https://user-images.githubusercontent.com/6311784/128868051-8dabd03c-728b-4a13-a543-d67015d45141.png)
Let's keep it capitalized